common/src/worldgen.rs

```diff
@@ -266,7 +266,8 @@ impl ChunkParams {
             let trilerp_coords = center.map(|x| (1.0 - x) * 0.5);
 
             let rain = trilerp(&self.env.rainfalls, trilerp_coords) + rng.sample(&normal.unwrap());
-            let temp = trilerp(&self.env.temperatures, trilerp_coords) + rng.sample(&normal.unwrap());
+            let temp =
+                trilerp(&self.env.temperatures, trilerp_coords) + rng.sample(&normal.unwrap());
```

Comment:
This too.

Reply:
However, this one is related to ignoring the consistency of individual commits assuming they would be squashed. I'll fix that.